### PR TITLE
Add translation for glossary

### PR DESCRIPTION
**Pull request by Manugal**

### CODE REVIEW

File: content/docs/reference-glossary.md [hunk 1]
```diff
@@ -7,37 +7,37 @@ permalink: docs/glossary.html
...
 
-## ES6, ES2015, ES2016, etc {#es6-es2015-es2016-etc}
+## ES6, ES2015, ES2016, ecc {#es6-es2015-es2016-etc}
 
-These acronyms all refer to the most recent versions of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. The ES6 version (also known as ES2015) includes many additions to the previous versions such as: arrow functions, classes, template literals, `let` and `const` statements. You can learn more about specific versions [here](https://en.wikipedia.org/wiki/ECMAScript#Versions).
+Tutti questi acronimi si riferiscono alle versioni più recenti dello standard ECMAScript Language Specification, di cui il linguaggio JavaScript ne è un'implementazione. La versione ES6 (nota anche come ES2015) include molte aggiunte rispetto alle versioni precedenti come: arrow functions, classi, stringhe template, istruzioni `let` e `const`. Puoi scoprire di più riguardo le versioni specifiche [qui](https://en.wikipedia.org/wiki/ECMAScript#Versions).
```
Comment (LucaBlackDragon) [inline content/docs/reference-glossary.md]:
```suggestion
Tutti questi acronimi si riferiscono alle versioni più recenti dello standard ECMAScript Language Specification, di cui il linguaggio JavaScript è un'implementazione. La versione ES6 (nota anche come ES2015) include molte aggiunte rispetto alle versioni precedenti, ad esempio: arrow functions, classi, stringhe template, istruzioni `let` e `const`. Puoi scoprire di più riguardo le versioni specifiche [qui](https://en.wikipedia.org/wiki/ECMAScript#Versions).
```

File: content/docs/reference-glossary.md [hunk 2]
```diff
@@ -7,37 +7,37 @@ permalink: docs/glossary.html
...
+Un compilatore JavaScript prende il codice JavaScript, lo trasforma e ritorna codice JavaScript in un formato differente. Il caso d'uso più comune è quello di prendere un codice con sintassi ES6 e trasformarlo in uno con sintassi che può essere interpretata da browser più vecchi. [Babel](https://babeljs.io/) è il compilatore più comunemente usato con React. 
 
 ## Bundlers {#bundlers}
 
-Bundlers take JavaScript and CSS code written as separate modules (often hundreds of them), and combine them together into a few files better optimized for the browsers. Some bundlers commonly used in React applications include [Webpack](https://webpack.js.org/) and [Browserify](http://browserify.org/).
+I bundlers prendono codice JavaScript e CSS scritto in moduli separati (spesso sono centinaia di essi), e li raggruppa in alcuni file meglio ottimizzati per i browser. Tra i bundler più comunemente usati nelle applicazioni React troviamo [Webpack](https://webpack.js.org/) e [Browserify](http://browserify.org/).
```
Comment (LucaBlackDragon) [inline content/docs/reference-glossary.md]:
```suggestion
I bundlers prendono codice JavaScript e CSS scritto in moduli separati (spesso sono centinaia) e li raggruppa in alcuni file meglio ottimizzati per i browser. Tra i bundler più comunemente usati nelle applicazioni React troviamo [Webpack](https://webpack.js.org/) e [Browserify](http://browserify.org/).
```

File: content/docs/reference-glossary.md [hunk 3]
```diff
@@ -7,37 +7,37 @@ permalink: docs/glossary.html
...
 
 ## CDN {#cdn}
 
-CDN stands for Content Delivery Network. CDNs deliver cached, static content from a network of servers across the globe. 
+CDN sta per Content Delivery Network. Le CDN consegnano contenuto statico e in cache da una rete di server sparsi nel globo.
 
 ## JSX {#jsx}
 
-JSX is a syntax extension to JavaScript. It is similar to a template language, but it has full power of JavaScript. JSX gets compiled to `React.createElement()` calls which return plain JavaScript objects called "React elements". To get a basic introduction to JSX [see the docs here](/docs/introducing-jsx.html) and find a more in-depth tutorial on JSX [here](/docs/jsx-in-depth.html).
+JSX è un'estensione sintattica di JavaScript. E' simile ad un linguaggio template, ma possiede tutta la potenza di JavaScript. JSX viene compilato in chiamate a `React.createElement()` che ritornano oggetti JavaScript chiamati "elementi React". Per un'introduzione base a JSX [vedi la documentazione qui](/docs/introducing-jsx.html) e puoi trovare un tutorial più dettagliato su JSX [qui](/docs/jsx-in-depth.html).
```
Comment (LucaBlackDragon) [inline content/docs/reference-glossary.md]:
```suggestion
JSX è un'estensione sintattica di JavaScript. E' simile ad un linguaggio template, ma possiede tutta la potenza di JavaScript. JSX viene compilato in chiamate a `React.createElement()` che ritornano oggetti JavaScript chiamati "elementi React". Per un'introduzione base a JSX [puoi leggere qui la documentazione](/docs/introducing-jsx.html) e trovare un tutorial più dettagliato su JSX [qui](/docs/jsx-in-depth.html).
```

File: content/docs/reference-glossary.md [hunk 4]
```diff
@@ -77,38 +77,38 @@ class Welcome extends React.Component {
...
 props.number = 42;
 ```
 
-If you need to modify some value in response to user input or a network response, use `state` instead.
+Se hai bisogno di modificare alcuni valori in risposta a input dell'utente o una risposta di rete, usa `state`.
 
 ### `props.children` {#propschildren}
 
-`props.children` is available on every component. It contains the content between the opening and closing tags of a component. For example:
+`props.children` è disponibile su ogni componente. Essa comprende il contenuto tra i tag di apertura e chiusura di un componente. Ad esempio:
```
Comment (LucaBlackDragon) [inline content/docs/reference-glossary.md]:
```suggestion
`props.children` è disponibile in ogni componente. Essa comprende il contenuto tra i tag di apertura e chiusura di un componente. Ad esempio:
```

File: content/docs/reference-glossary.md [hunk 5]
```diff
@@ -120,47 +120,47 @@ class Welcome extends React.Component {
...
+La differenza più importante tra `state` e `props` è che le `props` sono passate da un componente padre, ma lo `state` viene gestito dal componente stesso. Un componente non può cambiare le sue `props`, ma può cambiare il suo `state`. Per farlo, deve chiamare `this.setState()`. Solo i componenti definiti come classi possono avere uno state. 
 
-For each particular piece of changing data, there should be just one component that "owns" it in its state. Don't try to synchronize states of two different components. Instead, [lift it up](/docs/lifting-state-up.html) to their closest shared ancestor, and pass it down as props to both of them.
+Per ogni pezzo di dato che cambia, ci dovrebbe essere solo un componente che "possiede" tale dato nel suo state. Non provare a sincronizzare gli state di due componenti diversi. Invece, [sollevalo](/docs/lifting-state-up.html) verso l'antenato comune più vicino, e passalo come props ad entrambi.
```
Comment (LucaBlackDragon) [inline content/docs/reference-glossary.md]:
```suggestion
Per ogni pezzo di dato che cambia, ci dovrebbe essere solo un componente che "possiede" tale dato nel suo state. Non provare a sincronizzare lo state di due componenti diversi. Invece, [sollevalo](/docs/lifting-state-up.html) verso l'antenato comune più vicino, e passalo come props ad entrambi.
```

**Comment by deblasis** [comment]
Ciao @Manugal, do you have the chance to take a look at the suggestions? We are about to go live 😄

**Comment by Manugal** [comment]
Ciao.

Sorry about that, I forgot it.

**Pull request merged**